config/_default/config.yml

```diff
@@ -27,18 +27,6 @@ module:
           target: layouts
         - source: data
           target: data
-        - source: assets
-          target: assets
-        - source: archetypes
-          target: archetypes
-        - source: components
-          target: assets
-        - source: package.json
-          target: assets
-        - source: tsconfig.json
```

Comment:
For my benefit--can we remove these from `registry` here also, or is there a distinction?

Reply:
Yes! I'm about to push up a commit that does just that. (Just noticed them myself.)